src/main/java/frc/robot/commands/drive/DefaultDriveCommand.java

```diff
@@ -46,6 +46,10 @@ public void execute() {
         rotation = rotationSupplier.getAsDouble();
         trigger = triggerSupplier.getAsDouble();
         forceSlow = forceSlowSupplier.getAsBoolean();
+
+        if (x != 0 || y != 0) {
+            drivetrainSubsystem.setBalanceTarget(null);
```

Comment:
Lets set the translationTarget to null as well in case we ever decide to use the SetTranslationTarget command you implemented